docs/machineset-aws.md

```diff
@@ -0,0 +1,73 @@
+# Creating a Windows MachineSet
```

Comment:
I suggest `Create an AWS Windows MachineSet`.

Reply:
retitled to `Creating an AWS Windows MachineSet` (keeping the gerund, but adding the platform)